test/1.x/lazy-element.html

```diff
@@ -7,24 +7,31 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+
+<link rel="import" href="../../../polymer/polymer.html">
+
 <dom-module id="lazy-element-behavior">
   <template>
     <style>
       :host {
         border: 2px solid red;
       }
     </style>
+
     <slot></slot>
   </template>
+
   <script>
     Polymer({
       is: 'lazy-element-behavior',
+
       properties: {
         importLoaded: Boolean,
       },
+
       ready: function() {
         this.importLoaded = true;
-      },
+      }
```

Comment:
Okay, I just thought the Google preset required trailing comma. If not no worries